Poker.BE/Poker.BE.Domain/Game/Deck.cs

```diff
@@ -6,22 +6,28 @@
 
 namespace Poker.BE.Domain.Game
 {
-    public class Deck
+	/// <summary>Created Deck of Card and shuffle cards UC</summary>
+	/// <remarks>
+	/// <author>Gal Wainer</author>
+	/// <lastModified>2017-04-26</lastModified>
+	/// </remarks>
+	public class Deck
     {
 		#region Properties
-		List <Card> DeckOfCards;
+		protected List<Card> DeckOfCards { get; }
```

Comment:
Please use `IList<>` or `ICollection<>` and not `List<>` as the type of the field / Property. `List<>` is a specific implementation, not an interface.